src/shared_modules/content_manager/tests/unit/CtiApiDownloader_test.cpp

```diff
@@ -165,22 +124,72 @@ TEST_F(CtiApiDownloaderTest, TestHandleAnInvalidUrl)
     m_spUpdaterBaseContext->downloadsFolder = m_spUpdaterBaseContext->outputFolder / DOWNLOAD_FOLDER;
     m_spUpdaterBaseContext->contentsFolder = m_spUpdaterBaseContext->outputFolder / CONTENTS_FOLDER;
 
-    m_spUpdaterContext->spUpdaterBaseContext = m_spUpdaterBaseContext;
-
-    const auto expectedStageStatus = R"(
-        [
-            {
-                "stage": "CtiApiDownloader",
-                "status": "fail"
-            }
-        ]
-    )"_json;
-
     EXPECT_THROW(m_spCtiApiDownloader->handleRequest(m_spUpdaterContext), std::runtime_error);
 
     EXPECT_TRUE(m_spUpdaterContext->data.at("paths").empty());
 
     const auto stageStatus = m_spUpdaterContext->data.at("stageStatus");
 
-    EXPECT_EQ(stageStatus, expectedStageStatus);
+    EXPECT_EQ(stageStatus, FAIL_STATUS);
+}
+
+/**
+ * @brief Test the retry feature of the downloader when the server responses with 5xx errors.
```

Comment:
Typo:
```suggestion
 * @brief Test the retry feature of the downloader when the server responds with 5xx errors.
```
Also in the other test cases descriptions.

Reply:
Done.